.travis.yml

```diff
@@ -7,11 +7,3 @@ install:
  - ./.travis-setup.sh
 script:
  - mocha --timeout 20000 --globals Intl,IntlPolyfill,ErrorHandler test/tests.js
-notifications:
-  webhooks:
-    urls:
-      - YOUR_WEBHOOK_URL
```

Comment:
@dontcallmedom, @guibbs is not entirely sure this is a wrong copy and paste. I cannot find `YOUR_WEBHOOK_URL` set anywhere so, unless this is set in an environment variable, it looks very much like boilerplate code to me.
Do you confirm?